doc/ruleSets/SymfonyRisky.rst

```diff
@@ -2,7 +2,7 @@
 Rule set ``@Symfony:risky``
 ===========================
 
-Rules that follow the official `Symfony Coding Standards <https://symfony.com/doc/current/contributing/code/standards.html>`_ This set contains rules that are risky.
+Rules that follow the official `Symfony <https://symfony.com/doc/current/contributing/code/standards.html>`_ Coding Standards. This set contains rules that are risky.
```

Comment:
:-1:
we are actually linking to `Symfony Coding Standards`, not to Symfony

Reply:
:+1:  reverted